### PR TITLE
Add 1.1.1 to manual sidebar

### PR DESCRIPTION
**Pull request by mspncp**

### CODE REVIEW

File: inc/mansidebar.shtml [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <ul>
       <li><a href="/docs/manmaster">master</a></li>
       <li><a href="/docs/man1.1.0">1.1.0</a></li>
+      <li><a href="/docs/man1.1.1">1.1.1</a></li>
```
Comment (richsalz) [inline inc/mansidebar.shtml:7]: shouldn't this be before 1.1.0 ?

Comment (mspncp) [reply inc/mansidebar.shtml:7]: Oh, sure. That was my original intention :-). I will fix it.

Comment (mspncp) [reply inc/mansidebar.shtml:7]: Done.

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
Needs a second OMC review

**Review by t-j-h — APPROVED** [review]

**Comment by mattcaswell** [comment]
Pushed. Thanks.

**Comment by mspncp** [comment]
Matt, did this go online already? Strangely, the sidebar for [1.1.0](https://www.openssl.org/docs/man1.1.0) now shows all manual versions, while the sidebars for  [master](https://www.openssl.org/docs/manmaster),  [1.1.1](https://www.openssl.org/docs/man1.1.1) and [1.0.2](https://www.openssl.org/docs/man1.0.2) are still missing the link to version 1.1.1.

**Comment by mattcaswell** [comment]
Yes, it went online. I just purged the CDN cache manually which seems to have fixed some pages but not others. I suspect an issue in our purging script. Perhaps @levitte has an idea.

**Comment by levitte** [comment]
The docs directory is ignored by the `purge-one-hour` script, 'cause the purge program it uses gets overwhelmed by too many files, and simply didn't work with directories, at least a year ago, and I have frankly not looked at that since...

However, I just tried a simple trigger of `docs/manmaster/index.html`, to no avail...  so it seems to me the issues lies with the CDN, unless I'm doing something very wrong...

**Comment by mattcaswell** [comment]
This URL is fine:

https://www.openssl.org/docs/manmaster/index.html

But this one is not:

https://www.openssl.org/docs/manmaster

So it seems to be URLs which don't include an explicit ".html"

**Comment by levitte** [comment]
Oh ok, so the mechanism supports directories now...  separately.

**Comment by mattcaswell** [comment]
Hmmm, this works:

https://www.openssl.org/docs/manmaster/man3/index.html

But this still does not:

https://www.openssl.org/docs/manmaster/man3/

**Comment by levitte** [comment]
Hmmm, the purging script doesn't go into other docs trees than manmaster, wonder why...

**Comment by levitte** [comment]
Ah, because `-mtime -2`.  I can add a control to force all files to be invalidated.

**Comment by levitte** [comment]
Oh, no, it's because the script only look at included `.inc` files, not `.shtml` ones...

**Comment by richsalz** [comment]
Only actual files, not directories which get auto-indexed.

**Comment by levitte** [comment]
Okie

**Comment by levitte** [comment]
Either way, I've done quite a bit of fixing of the main purging script, it's become better at detecting the .html files that are affected by diverse changes (including in `inc/`).

*[going back to vacationing]*

**Comment by mattcaswell** [comment]
Seems to be working now. Thanks @levitte.

**Comment by mspncp** [comment]
Thanks from me, too!

**Pull request closed without merging**